test/recommend/test_recommend_pipeline.py

```diff
@@ -175,7 +177,7 @@ def test_text_format(self):
         new_mock = format_converter_for_pattern(mock_input)
         text = create_text(new_mock, full_report=True)
         md5_hash = md5('\n'.join(text).encode('utf-8'))
-        self.assertEqual(md5_hash.hexdigest(), '5efdc7930a66874e9d0c7dcfef734687')
+        self.assertEqual(md5_hash.hexdigest(), '2a67e22091ba2cfd76847d292ed90142')
```

Comment:
why is hash changed?

Reply:
Error occured was replaced with Error occurred